fast64_internal/f3d/f3d_writer.py

```diff
@@ -554,7 +555,16 @@ def saveStaticModel(
             )
 
     for drawLayer, fMesh in fMeshes.items():
-        if revertMatAtEnd:
+        #inline requires the mesh to revert a few cmds for safety
+        if fMesh.inline:
+            # basic revert adds one too many syncs for inline
+            revertMatAndEndDraw(tmpList := GfxList("", GfxListTag.Draw, DLFormat.Static), [DPSetRenderMode(fModel.getRenderMode(drawLayer), None), DPSetCycleType("G_CYC_1CYCLE"), DPSetTextureLUT("G_TT_NONE")])
+            tmpList.commands.remove(DPPipeSync())
+            
+            fModel.endDraw(fMesh, obj)
+            # add revert list without sync
+            fMesh.draw.commands.extend(tmpList.commands)
```

Comment:
Honestly, I wouldn't even use that function at all in this case, since its small enough and you're modifying its output anyway. Or it can be moved into f3d_gbi.